hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/impl/HeaderProcessing.java

```diff
@@ -325,19 +325,6 @@ private Map<String, byte[]> retrieveHeaders(
         md.contentEncoding());
     maybeSetHeader(headers, XA_CONTENT_LANGUAGE,
         md.contentLanguage());
-    // If CSE is enabled, use the unencrypted content length.
```

Comment:
uncomment?

Reply:
This is not required since it is already handled in getObjectMetadata call